### ISSUE TITLE
we should add a contact page

### ISSUE THREAD

**Issue by robertoostenveld** [Radboud-University/osc-nijmegen#10]
allowing people to subscribe themselves to our email list.

**Comment by robertoostenveld** [comment]
The first option that I encountered after a quick search was https://formspree.io, which seems easy to use. It requires a small piece of static html to be inserted in a page.

**Comment by KristijanArmeni** [comment]
Thanks. @Frederik-D-Weber your domain registration service also had an option of email, right? I can try to use the above form service by setting `action="https://formspree.io/contact@openscience-nijmegen.nl"`. What client would be used to check the received forms/emails?

**Comment by robertoostenveld** [comment]
I don't think that initially any client is needed to parse the email forms. You would just do that yourself and copy-paste the email address of the person that contacts you over to some list that you store on a safe location (wrt GDPR).

**Comment by robertoostenveld** [comment]
Oh, I realize now that with "client" you probably mean which email client, i.e. software or website to read the emails.

**Comment by Frederik-D-Weber** [comment]
created a full email address contact@openscience-nijmegen.nl with POP3/IMAP and secure SMTP server or use over webinterface and with an automatic reply message.
@KristijanArmeni will handle first, has full access.

**Comment by KristijanArmeni** [comment]
I added a simple form that collects only user's e-mail as text for input. I created a separate page, where users can submit unsubscribe form by providing again their e-mail as text input. This page can be linked in e-mails that will be send around.

subscribe form: https://openscience-nijmegen.nl/_pages/contact/
ubsubscribe form: https://openscience-nijmegen.nl/_pages/unsubscribe/

@robertoostenveld I believe this puts the most basic functionality we discussed in place. Pretty straighforward for the user, a little bit of manual work for the e-mail manager. Any ideas for a good way of securely storing the list? An encrypted file on a local drive?
